fusedrug/utils/misc.py

```diff
@@ -15,5 +15,12 @@ def worker_init_fn(worker_id: int) -> None:
     # to ensure different seed on each worker
```

Comment:
I think we have all of this in fuse-med-ml already

Reply:
thanks - I can see the class Seed in FuseMedML seed.py
It's not identical, I'll compare and potentially just use the one in FuseMedML and/or modify it